charts/spire/charts/spire-server/values.yaml

```diff
@@ -184,6 +183,25 @@ upstreamAuthority:
         certificate: ""
         key: ""
         bundle: ""
+  awsPCA:
+    enabled: true
+    # -- AWS Region to use
+    region: "us-west-2"
+    # -- ARN of the "upstream" CA certificate
+    certificateAuthorityArn: "arn:aws:acm-pca:us-west-2:945673678728:certificate-authority/636dbcd8-203f-43e0-9701-be854b094a2b"
```

Comment:
ARN instead of Arn please, since its an acronym. Same throughout.

Reply:
addressed in the new commit